code/addons/docs/package.json

```diff
@@ -98,15 +98,14 @@
     "prep": "node --loader ../../../scripts/node_modules/esbuild-register/loader.js -r ../../../scripts/node_modules/esbuild-register/register.js ../../../scripts/prepare/bundle.ts"
   },
   "dependencies": {
-    "@jest/transform": "^29.3.1",
-    "@mdx-js/react": "^2.1.5",
+    "@babel/core": "^7.12.3",
```

Comment:
Do you have any substantial reason why not the latest version of @babel/core is used?

Reply:
This was just the version defined in mdx2-csf. The caret should resolve to the latest version though. I can upgrade as well.